src/app/metadata/features/dataset-details/dataset-details.component.html

```diff
@@ -144,7 +144,7 @@ <h4 class="inline-flex items-center">
                 </p>
                 @if (pub.abstract) {
                   @for (
-                    par of pub.abstract.split('\\n');
+                    par of pub.abstract | newlineSplit;
```

Comment:
Can we create a shared UI component for this whole construct (for loop that emits paragraphs) and reuse it in all of these places, like `<app-paragraphs title="Policy">{{pub.abstract}}</app-paragraphs>`?